config/config.example.env

```diff
@@ -0,0 +1,9 @@
+# Default development
```

Comment:
- [ ] Can we use `.env` instead and set all the default values so people do not need to configure this themselves?